org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Antares/ImportAntaresImpl.java

```diff
@@ -61,32 +61,32 @@ public static boolean canLoadThisStream(final InputStream is) {
 	 * Method that import the Antares format given through a InputStream into the
 	 * Layers.
 	 *
-	 * @param _is        InputStream where the Antares format is available.
-	 * @param _layers    New Track will be inserted into this layers
-	 * @param _trackName Name of the new track (since it is not available in the
+	 * @param is        InputStream where the Antares format is available.
+	 * @param layers    New Track will be inserted into this layers
+	 * @param trackName Name of the new track (since it is not available in the
 	 *                   Antares format)
-	 * @param _month     Month to use in the new track (since it is not available in
+	 * @param month     Month to use in the new track (since it is not available in
 	 *                   the Antares format)
-	 * @param _year      Year to use in the new track (since it is not available in
+	 * @param year      Year to use in the new track (since it is not available in
 	 *                   the Antares format)
 	 * @return A list with the errors found while reading the Antares format.
 	 */
-	public static List<ImportAntaresException> importThis(final InputStream _is, final Layers _layers,
-			final String _trackName, final int _month, final int _year) {
+	public static List<ImportAntaresException> importThis(final InputStream is, final Layers layers,
```

Comment:
@saulhidalgoaular - I don't really remember how/when I started it, but I just use leading underscore for class variables, as described here: https://stackoverflow.com/a/20677292/92441